lib/drivers/docker/index.js

```diff
@@ -392,3 +419,9 @@ DockerDriver.prototype.getName = function() {
 DockerDriver.prototype.getStatus = function() {
   return 'running';
 };
+
+function publicIp() {
+  var eths = os.networkInterfaces();
+  var addrs = eths.en0 || eths.eth0;
```

Comment:
This runs on the host system right? Seems to be too restrictive. How about search for the first IPv4 addr which is not in the 127.* space.